### PR TITLE
Add 'first' instead of '1st' for referees + change heading size

### PR DESCRIPTION
**Pull request by willmcb**
## Context

Fixes some discrepancy between app and design.

Referee headings were coming up as '1st reference', '2nd reference' this PR changes this so they read 'First referee', 'Second referee' etc. 

More details in the trello card.

Also changes the `<h2>` tags to govuk mediums rather than large.

## Changes proposed in this pull request

**Before:**
image

**After**
Screenshot 2020-03-06 at 12 12 55

## Link to Trello card

https://trello.com/c/tU8HFZco/1732-fix-referee-sizing-and-content

## Things to check

- [x] This code doesn't rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/services/text_ordinalizer.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  end
+
+  class << self
+  private
+
+    def text_ordinalize(value)
+      ordinalize_mapping[value] || value.ordinalize
+    end
+
+    def ordinalize_mapping
```
Comment (stevehook) [inline app/services/text_ordinalizer.rb]: Maybe this should be a constant rather than a method?

Comment (stevehook) [reply app/services/text_ordinalizer.rb]: You could argue that it should be in the locale yml files but based on the conversation we had the other week I think we are best off avoiding that.

**Comment by fofr** [comment]
Thank you 🙏

File: app/services/text_ordinalizer.rb (hunk 1 above)
Comment (willmcb) [reply app/services/text_ordinalizer.rb]: Have constantised it 👍

**Pull request merged**